genki_wave/protocols.py

```diff
@@ -69,6 +70,7 @@ class ProtocolAsyncio(ProtocolAbc, Packetizer):
 
     def __init__(self):
         super().__init__()
+        get_or_create_event_loop()
```

Comment:
Creating an `asyncio.Queue` object is only possible if there is some event loop active, this hasn't surfaced before because we have only ever called this code from the main thread